README.md

```diff
@@ -52,14 +56,17 @@ See the [Hello, World!](examples/hello/lib/app.dart) demo in the
     [CloudEvents](https://cloudevents.io/) spec
 *   Portable between serverless platforms
 
-## Quickstart: Hello, World on your local machine
+## Quickstart
+
+"Hello, World" on your local machine.
 
 ```shell
+$ cd examples/hello
```

Comment:
Don't include the `$`, such that folks can copy-paste.

Put CLI input and output in separate shell snippets:

```shell
cd examples/hello
docker build -t app .
docker run -it -p 8080:8080 --name demo app
```

This will print out `Listening on :8080`.